apps/server/src/modules/room-membership/service/room-membership.service.ts

```diff
@@ -20,19 +20,15 @@ export class RoomMembershipService {
 		private readonly userService: UserService
 	) {}
 
-	private async createNewRoomMembership(
-		roomId: EntityId,
-		userId: EntityId,
-		roleName: RoleName.ROOMEDITOR | RoleName.ROOMVIEWER
-	): Promise<RoomMembership> {
+	public async createNewRoomMembership(roomId: EntityId, ownerUserId: EntityId): Promise<RoomMembership> {
 		const room = await this.roomService.getSingleRoom(roomId);
 
 		const group = await this.groupService.createGroup(
 			`Room Members for Room ${roomId}`,
 			GroupTypes.ROOM,
 			room.schoolId
 		);
-		await this.groupService.addUsersToGroup(group.id, [{ userId, roleName }]);
+		await this.groupService.addUsersToGroup(group.id, [{ userId: ownerUserId, roleName: RoleName.ROOMOWNER }]);
```

Comment:
Does that mean that we can more than one owner of a room?

Reply:
no we cant. The group itself doesnt mind, since it doesnt know anything about rooms. But the roomMembership Service itself in its `addMembersToRoom` function doesnt accept owner as a role. Currently, the only way an owner is added is during creation of the RoomMembership, later there will also be the possability to pass the ownership on to a different person, while at the same time demoting the old owner.